composer.json

```diff
@@ -22,7 +22,7 @@
     "source": "https://github.com/getkirby/starterkit"
   },
   "require": {
-    "php": ">=8.0.0 <8.2.0",
+    "php": ">=8.0.0 <8.2.2",
```

Comment:
```suggestion
    "php": ">=8.0.0 <8.3.0",
```

Reply:
php 8.3 is currently only a RC and will take a while to be finally released, right? 
But I will adapt it if you want me to :)

Reply:
Check out our `kirby` repository:
https://github.com/getkirby/kirby/blob/main/composer.json#L27

Reply:
> Check out our `kirby` repository: https://github.com/getkirby/kirby/blob/main/composer.json#L27

yeah okay. makes sense to align it here then too